packages/js/src/utils/id-token.ts

```diff
@@ -24,3 +29,23 @@ export const decodeIdToken = (token: string): IdTokenClaims => {
   const json = UrlSafeBase64.decode(encodedPayload);
   return s.create(JSON.parse(json), IdTokenClaimsSchema);
 };
+
+/**
+ * Verify ID Token
+ *
+ * @param idToken
+ * @param clientId
+ * @param issuer
+ * @param jwks
+ */
+export const verifyIdToken = async (
+  idToken: string,
+  clientId: string,
+  issuer: string,
+  jwks: JWTVerifyGetKey
+) => {
+  const result = await jwtVerify(idToken, jwks, { audience: clientId, issuer });
+  if (Math.abs((result?.payload?.iat ?? 0) - Date.now() / 1000) > issuedAtTimeTolerance) {
+    throw new IdTokenVerifyError('invalid issued at time');
```

Comment:
the error code should be a constant / enum. please refer to `RequestError` in @logto-io/core

Reply:
Added `LogtoError` with specific code, PTAL